packages/itwinui-css/backstop/tests/non-ideal-state.html

```diff
@@ -30,7 +30,7 @@
 
       .demo-non-ideal-state-scrolling {
         height: 100vh;
-        scroll-snap-align: start;
+        scroll-snap-align: flex-start;
```

Comment:
flex-start didn't work here when I tested it. Maybe can replace with `center` ([Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-snap-align#syntax)).
```suggestion
        scroll-snap-align: center;
```
Also, to confirm, the warning in [#824](https://github.com/iTwin/iTwinUI/issues/824) is for slider.css, so maybe this change is unneeded.

Reply:
I can undo this change.